templates/base.html

```diff
@@ -28,16 +28,16 @@ <h1><a href= "/" style="text-decoration: none; color: inherit;">Latin Scansion E
+					<p>The grammars themselves were created using <a href="https://thrax.opengrm.org/">Thrax</a>, a domain-specific language (and associated compiler) for developing finite-state grammars. Our own <span class="tt"><a href="https://github.com/CUNY-CL/latin_scansion">latin_scansion</a></span> library acts as a backend. It applies the Thrax grammars to the input using <a href=https://pynini.opengrm.org>Pynini</a>, a Python extension library for finite-state grammar development, and outputs a custom <a href="https://developers.google.com/protocol-buffers/">protocol buffer</a> containing the scan and phonetic transcription. The <a href="https://flask.palletsprojects.com/">Flask</a> web framework and the <a href="https://jinja.palletsprojects.com/">Jinja</a> template engine are then used to render the scansion/transcription protocol buffer. The site itself is served by the <a href="https://gunicorn.org/">Gunicorn</a> web server.</p>
 					<p>The finite-state grammars process the input strings in the following order:
 						<ol>
-							<li><tt><a href="https://github.com/CUNY-CL/latin_scansion/blob/master/grammars/normalize.grm">normalize.grm</a></tt> removes punctuation and case-folds.</li>
-							<li><tt><a href="https://github.com/CUNY-CL/latin_scansion/blob/master/grammars/pronounce.grm">pronounce.grm</a></tt> applies Latin grapheme-to-phoneme rules to obtain a broad phonetic transcription.</li>
-							<li><tt><a href="https://github.com/CUNY-CL/latin_scansion/blob/master/grammars/variable.grm">variable.grm</a></tt> optionally applies various phonetic rules associated with scansion (rules such as elision and resyllabification). These variable rules are all associated with non-zero weights and so they are only applied when needed to "make the meter".</li>
-							<li><tt><a href="https://github.com/CUNY-CL/latin_scansion/blob/master/grammars/syllable.grm">syllable.grm</a></tt> syllabifies the phonetic transcriptions and subdivides into onset, nucleus, and coda.</li>
-							<li><tt><a href="https://github.com/CUNY-CL/latin_scansion/blob/master/grammars/weight.grm">weight.grm</a></tt> codes syllables as heavy or light.</li>
-							<li><tt><a href="https://github.com/CUNY-CL/latin_scansion/blob/master/grammars/foot.grm">foot.grm</a></tt> groups sequences of heavy and light syllables into dactyls, spondees, and trochees.</li>
-							<li><tt><a href="https://github.com/CUNY-CL/latin_scansion/blob/master/grammars/hexameter.grm">hexameter.grm</a></tt> filters foot sequences to only those permitted in a hexameter verse.</li>
+							<li><span class="tt"><a href="https://github.com/CUNY-CL/latin_scansion/blob/master/grammars/normalize.grm">normalize.grm</a></span> removes punctuation and case-folds.</li>
```

Comment:
The tt tag is depreciated from HTML 5 and no longer recommended. I used CSS instead to create the same look.

Reply:
Oh I had no idea. Thanks for sussing that out. (In this case I generated the HTML using a Markdown converter.)